content/actions/reference/workflow-commands-for-github-actions.md

```diff
@@ -210,22 +210,35 @@ echo "::add-mask::$MY_NAME"
 
 Stops processing any workflow commands. This special command allows you to log anything without accidentally running a workflow command. For example, you could stop logging to output an entire script that has comments.
 
-### Example stopping workflow commands
+{% warning %}
 
-``` bash
-echo "::stop-commands::pause-logging"
-```
+**Warning:** Make sure the token you're using is randomly generated and different between runs. A **hash** of your `github.token` is a good choice for this.
+
+{% endwarning %}
 
 To start workflow commands, pass the token that you used to stop workflow commands.
 
 `::{endtoken}::`
 
-### Example starting workflow commands
+### Example stopping and starting workflow commands
 
-``` bash
-echo "::pause-logging::"
+{% raw %}
+
+```yaml
+workflow-command-job:
```

Comment:
Technically fine but looks a bit off to use four spaces indent for this level and two for all others.

This workflow is valid, but I don't know about the workflow commands so can't sign off on that 😬 

Maybe emphasize that this is a job only and should be inserted into (and further indented to fit into) a workflow file `jobs` section.

Reply:
👋 Thanks @anywherepilot - I've updated the YAML in this commit: https://github.com/github/docs/pull/9426/commits/6479a7ffa6f3819c65dffd2bb4fcb48fdcbd3e26